server/etcdmain/config.go

```diff
@@ -291,6 +291,7 @@ func newConfig() *config {
 
 	// unsafe
 	fs.BoolVar(&cfg.ec.UnsafeNoFsync, "unsafe-no-fsync", false, "Disables fsync, unsafe, will cause data loss.")
+	fs.BoolVar(&cfg.ec.UnsafeAllowClusterVersionDowngrade, "unsafe-allow-cluster-version-downgrade", cfg.ec.UnsafeAllowClusterVersionDowngrade, "true to allow cluster version downgrade, because newer minor versions may introduce incompatible feature changes like lease checkpointer introduced in v3.4")
```

Comment:
I meant define some default value in `embed` package around here?

https://github.com/etcd-io/etcd/blob/b29e8e47a6e8b7a4017790e77a76be2691a652c9/server/embed/config.go#L85-L91

It can be something like

```
embed.DefaultUnsafeAllowClusterVersionDowngrade
```

Reply:
got it. Done with the latest version.